### PR TITLE
DD-1076: Create script update-thematische-collectie-csv.py

### PR DESCRIPTION
**Pull request by jo-pol**
Fixes DD-1076: Create script update-thematische-collectie-csv.py

# Description of changes

# How to test
* `poetry run update-thematische-collecties --help`
* this creates `.easy-migration-tools.yml` and tells you (on stderr) it did
* adjust fedora parameters and logging
* follow help instructions

# Related PRs

(Add links)

* https://github.com/DANS-KNAW/easy-dtap/pull/564

# Notify

@DANS-KNAW/dataversedans

### CODE REVIEW

File: src/easymigration/config.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+configuration_file = '.easy-migration-tools.yml'
+example_configuration_file = 'example-easy-migration-tools.yml'
+configuration_file_locations = [configuration_file, os.path.expanduser('~/' + configuration_file)]
+
+
+def ensure_configuration_file_exists():
+    config = find_config_file()
+    if config is None:
+        print("No %s found; instantiating in current directory" % configuration_file)
```
Comment (lindareijnhoudt) [inline src/easymigration/config.py:16]: why `print` instead of `logging`?

File: src/easymigration/scripts/update_thematische_collecties.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+from requests.auth import HTTPBasicAuth
+
+from easymigration.config import init
+
+
+def update_thematische_collecties(fedora_config):
+    """
+     Copies a Thematische-Collecties.csv from stdin to stdout.
+     Assumed columns: -,EASY-dataset-id,-,-,members
+     Empty members columns are filled by crawling jump of pages.
```
Comment (lindareijnhoudt) [inline src/easymigration/scripts/update_thematische_collecties.py]:
```suggestion
     Empty members columns are filled by crawling jump off pages.
```

File: src/easymigration/scripts/update_thematische_collecties.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    base_url = fedora_config["base_url"]
+    risearch_url = "{}/risearch".format(base_url)
+
+    csv_reader = csv.reader(sys.stdin, delimiter=',')
+    csv_writer = csv.writer(sys.stdout, delimiter=',')
+    csv_writer.writerow(next(csv_reader))
+
+    for row in csv_reader:
+        if not ("easy-dataset" in row[4]):
+            logging.debug("{} {} {}".format(row[1], row[4], row[5]))
```
Comment (lindareijnhoudt) [inline src/easymigration/scripts/update_thematische_collecties.py]: you can also read the csv with named columns. makes reading it a bit easier

File: src/easymigration/config.py (hunk 1 above)
Comment (jo-pol) [reply src/easymigration/config.py:16]: Copied from dans-datastation-tools. Logging is initialized by the content of this file. But `stderr` may be better than `stdout`.

**Pull request merged**